src/gvpn_controller.py

```diff
@@ -1,22 +1,23 @@
 #!/usr/bin/env python
 
 from ipoplib import *
-
+# GvpnUdpServer inherits the UdpServer class of the core IPOP Library
```

Comment:
First of all, sorry for all the emails, everyone. I hate that github does that for every inline comment. @GingerNinja23 sorry if I seem like I'm being harsh, I'm just trying to be thorough the first time around.

I think this comment is extraneous, because you're just converting the line below

```python
class GvpnUdpServer(UdpServer):
```

to English without adding anything for people that already understand python.

Instead, it may be good to put a module-level docstring (not a normal comment, see my later remarks) at the top of this file that gives a blurb about what GVPN actually is. You should be able to steal something from the ipop website for this.